Database/Corrections/sodQuestFixes.lua

```diff
@@ -3180,6 +3180,54 @@ function SeasonOfDiscovery:LoadQuests()
             [questKeys.requiredSpell] = -410095,
             [questKeys.zoneOrSort] = sortKeys.SHAMAN,
         },
+        [90206] = {
+            [questKeys.name] = "Carve",
+            [questKeys.startedBy] = {{1115,1116,1117,1118,208711}},
+            [questKeys.finishedBy] = nil,
+            [questKeys.requiredLevel] = 1,
+            [questKeys.questLevel] = 9,
+            [questKeys.requiredRaces] = raceIDs.NONE,
+            [questKeys.requiredClasses] = classIDs.HUNTER,
+            [questKeys.objectivesText] = {"Kill Rockjaw Troggs until Rabbit Musk drops, consume it and use Tame Beast on a Rabbit critter, then speak to Toby at Amberstill Ranch to receive the rune."},
+            [questKeys.requiredSpell] = -425758,
+            [questKeys.zoneOrSort] = sortKeys.HUNTER,
+        },
+        [90207] = {
+            [questKeys.name] = "Carve",
+            [questKeys.startedBy] = {{2009,2010,2011,2012,2013,2014,2152,209948}},
+            [questKeys.finishedBy] = nil,
+            [questKeys.requiredLevel] = 1,
+            [questKeys.questLevel] = 8,
+            [questKeys.requiredRaces] = raceIDs.NONE,
+            [questKeys.requiredClasses] = classIDs.HUNTER,
+            [questKeys.objectivesText] = {"Kill Gnarlpine Furbolgs until Deer Musk drops, consume it and use Tame Beast on a Deer critter, then speak to Raelaeron next to the hunter trainer in Darnassus to receive the rune."},
+            [questKeys.requiredSpell] = -425758,
+            [questKeys.zoneOrSort] = sortKeys.HUNTER,
+        },
+        [90208] = {
+            [questKeys.name] = "Carve",
+            [questKeys.startedBy] = {{3111,3114,3113,3112,5824,5826,208184}},
+            [questKeys.finishedBy] = nil,
+            [questKeys.requiredLevel] = 1,
+            [questKeys.questLevel] = 8,
+            [questKeys.requiredRaces] = raceIDs.NONE,
+            [questKeys.requiredClasses] = classIDs.HUNTER,
+            [questKeys.objectivesText] = {"Kill Razermane Quillboars until Adder Pheromone drops, consume it and use Tame Beast on a Adder critter, then speak to Razzil in Razor Hill to receive the rune."},
+            [questKeys.requiredSpell] = -425758,
+            [questKeys.zoneOrSort] = sortKeys.HUNTER,
+        },
+        [90209] = {
+            [questKeys.name] = "Carve",
+            [questKeys.startedBy] = {{2949,2950,2951,207515,205635}},
```

Comment:
We should also cleanup the spawns of `205635` and not show [Lurkmane](https://www.wowhead.com/classic/npc=207515/lurkmane) as a source of the musk.

```suggestion
            [questKeys.startedBy] = {{2949,2950,2951,205635}},
```